components/gripper/client_test.go

```diff
@@ -145,6 +148,9 @@ func TestClient(t *testing.T) {
 		test.That(t, err, test.ShouldNotBeNil)
 		test.That(t, err.Error(), test.ShouldContainSubstring, errStopUnimplemented.Error())
 
+		_, err = client2.Geometries(context.Background(), extra)
+		test.That(t, err.Error(), test.ShouldContainSubstring, gripper.ErrGeometriesNil(failGripperName).Error())
```

Comment:
would` Errors.Is ` work here? I think we are trying to avoid using substring for errors test when possible

Reply:
the problem is that it adds `"rpc error: code = Unknown desc = "` before the actual error, so figured it was easier to use substring. in the server code it doesn't add that string,  so you can just use `test.ShouldBeError`

Reply:
although I guess I didn't try `Errors.Is` perhaps that's easier, I've never seen that in the `test.That()` format, I'll try it out